components/cdc/src/endpoint.rs

```diff
@@ -950,7 +950,7 @@ impl<T: 'static + RaftStoreRouter<E>, E: KvEngine> Endpoint<T, E> {
                     resolved_regions.push(region_id, resolved_ts.min());
                     // The judge of raw region is not accuracy here, and we may miss at most one
                     // "normal" raw region. But this will not break the correctness of outlier detection.
-                    if resolved_ts.is_min_ts_from_raw() {
+                    if resolved_ts.is_min_ts_from_raw() || delegate.is_raw_region() {
```

Comment:
Suggest to add test case for the scenario that `txn_ts == raw_ts`.

Reply:
Added in `test_raw_dead_lock` test case.